src/System.ComponentModel.Primitives/src/System/ComponentModel/ISite.cs

```diff
@@ -2,51 +2,42 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System.Runtime.InteropServices;
-
 namespace System.ComponentModel
 {
-    /*
-     * Containers use sites to manage and communicate their child components.
-     *
-     * A site is a convenient place for a container to store container-specific
-     * per-component information.  The canonical example of such a piece of
-     * information is the name of the component.
-     *
-     * To be a site, a class implements the ISite interface.
-     */
-
     /// <summary>
-    ///    <para>Provides functionality required by sites. Sites bind
-    ///       a <see cref='System.ComponentModel.Component'/> to a <see cref='System.ComponentModel.IContainer'/>
-    ///       and enable communication between them, as well as provide a way
-    ///       for the container to manage its components.</para>
+    /// Containers use sites to manage and communicate their child components.
+    /// 
+    /// A site is a convenient place for a container to store container-specific
+    /// per-component information. The canonical example of such a piece of
+    /// information is the name of the component.
+    /// 
+    /// Provides functionality required by sites. Sites bind a
+    /// <see cref='System.ComponentModel.Component'/> to a
+    /// <see cref='System.ComponentModel.IContainer'/> and enable communication between
+    /// them, as well as provide a way for the container to manage its components.
```

Comment:
All of this extra content should be in a remark rather than summary.

Reply:
Also, newlines do not separate paragraphs in documentation comments. You'll want to wrap each paragraph of the `<remarks>` in `<para>content...</para>`.